File: bee_1066.py
def main():
    numeros_pares = 0
    numeros_impares = 0
    numeros_positivos = 0
    numeros_negativos = 0

    for i in range(5):
        numero = int(input())
        if (numero % 2 == 0):
            numeros_pares += 1
        else:
            numeros_impares += 1
        if (numero > 0):
            numeros_positivos += 1
        elif numero < 0:
            numeros_negativos += 1
        
    print(f"{numeros_pares} valor(es) par(es)")
    print(f"{numeros_impares} valor(es) impar(es)")
    print(f"{numeros_positivos} valor(es) positivo(s)")
    print(f"{numeros_negativos} valor(es) negativo(s)")
    
main()

File: bee_1066.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function main() {
    let numerosPositivos = 0;
    let numerosNegativos = 0;
    let numerosPares = 0;
    let numerosImpares = 0;
    for (let i = 0; i < 5; i++) {
        let number = Number(lines.shift());
        if (number % 2 == 0) {
            numerosPares++;
        } else {
            numerosImpares++;
        }
        if (number > 0) {
            numerosPositivos++;
        } else if (number < 0) {
            numerosNegativos++;
        }
    }

    console.log(numerosPares + " valor(es) par(es)");
    console.log(numerosImpares + " valor(es) impar(es)");
    console.log(numerosPositivos + " valor(es) positivo(s)");
    console.log(numerosNegativos + " valor(es) negativo(s)");
}

main()